setup.py

```diff
@@ -100,6 +100,7 @@
     CYTHON_DEPENDENCY,
     'packaging>=21.0',
     'setuptools-rust~=0.12.1',
+    'wheel~=0.37.0',  # needed by PyYAML and immutables, refs pypa/pip#5865
```

Comment:
Ugh. This could be a problem in edgedb-pkg too.  Could you elaborate on how it fails, please?

Reply:
`wheel` and `Cython` are in the [build `requires` of `PyYAML`](https://github.com/yaml/pyyaml/blob/master/pyproject.toml), and `pip download` won't fetch them at all. Because `PyYAML` hasn't provided a wheel release for Python 3.10, installing from source tarball will fail in the CI here:

https://github.com/edgedb/edgedb/blob/cb61a47469c282e2219875747dd3257d528adfc5/.github/workflows/tests.yml#L46-L47

Same for `immutables` but only `wheel` is needed.

Reply:
Oh, so I'd remove the version pin from `wheel` then.